src/main/webapp/resources/js/validation-date.js

```diff
@@ -0,0 +1,28 @@
+function validateBookingsDate() {
+
+    var fromDateValue = $('#from').val();
+    var toDateValue = $('#to').val();
+    var regex = /^(\d{4})(\/|-)(\d{1,2})(\/|-)(\d{1,2})$/;
+    var isValidate = true;
+
+    if (fromDateValue.match(regex) == null) {
+        nowYear = new Date().getFullYear();
+        month = fromDateValue.split("-")[1];
+        day = fromDateValue.split("-")[2];
+        $('#from').val(nowYear + "-" + month + "-" + day);
+        isValidate = false;
+    } else if (toDateValue.match(regex) == null) {
+        nowYear = new Date().getFullYear();
+        month = toDateValue.split("-")[1];
+        day = toDateValue.split("-")[2];
+        $('#to').val(nowYear + "-" + month + "-" + day);
```

Comment:
if so...

nowYear = new Date().getFullYear();    <- returns current year
month = fromDateValue.split("-")[1];    <- returns value of month that user wrote?!? (in case it's possible to split)
day = fromDateValue.split("-")[2];  <- returns value of day that user wrote?!?

Seems like, if data is not valid, then you say to user that it's not valid, and then set that field with InvalidData to (currentYear-monthUserWrotePreviously-dayUserWrotePreviously). 

But is it possible that you cant split to\forDateValue in case it's writing by user as String? and in case user setData through kind of DataPicker why we need to check if data is maches dataPatern at all?